mixins/localize-mixin.md

```diff
@@ -121,6 +121,65 @@ If your localized string contains arguments, pass them as a key-value object as
 
 ```javascript
 render() {
-  return html`<p>${this.localize('hello', {firstName: 'Mary'})}</p>`;
+  return html`<p>${this.localize('hello', { firstName: 'Mary' })}</p>`;
 }
 ```
+
+## `localizeHTML()`
+
+You can include some basic markup in your localization resources and safely convert them to HTML with the `localizeHTML()` method.
+
+### Simple Markup
+
+#### Bold
+
+The `[b]bold[/b]` markup converts to the `<strong>` tag.
+
+#### Italic
+
+The `[i]italic[/i]` markup converts to the `<em>` tag.
+
+#### Line Breaks
+
+The `[br]` markup converts to the `<br>` tag.
+
+### Complex Markup
+
+For more complex markup, additional information must be provided as the second parameter.
+
+#### Links
+
+The `[a]link[/a]` markup converts to the `<d2l-link>` tag, and takes an attributes string in the `link` property.
+
+> Return [a]home[/a].
+
+```javascript
+this.localizeHTML('termName', { link: 'href="home.html"' });
+```
+
+If a localization resource contains multiple links, give them unique identifiers and provide attributes from the `links` property instead:
+
+> Go [a home]home[/a] or [a back]return back[/a] to the previous page.
+
+```javascript
+this.localizeHTML('termName', {
+  links: {
+    home: 'href="home.html"',
+    back: 'href="back.html"'
+  }
+});
+```
+
+#### Help Tooltips
+
+The `[tooltip-help]help tooltip[/tooltip-help]` markup converts to the `<d2l-tooltip-help>` tag, and takes tooltip text in the `tooltipHelp` property.
```

Comment:
Would nesting work? For example: `[tooltip-help][b]help[/b] tooltip[/tooltip-help]`.

Worth noting that tooltips cannot contain complex semantics, and we force text semantics to ensure screen readers don't discard the content entirely.

Reply:
A simpler nesting is tested [here](https://github.com/BrightspaceUI/core/pull/3211/files#diff-516e261002a7e8e2dd6f3717ecdd945ef45431ce53e2979e666a54cf0cdaf5c9R132), but I think they should all work.

Reply:
And by "work", I mean not break. Your example above should output whatever `<d2l-tooltip-help text="<strong>help</strong> tooltip">...</d2l-tooltip-help>` does, which I assume doesn't include actual bolded text.